get-started-building-a-nuxt-3-blog-with-directus/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Build a blog with Nuxt 3 & Directus
+date: 2022-04-07
+author: Conner Bachmann
+image: ./build-a-blog-with-nuxt3-and-directus.webp
+category: integration
+---
+
+# Build a blog with Nuxt 3 & Directus
+
+> This guide shows you how to build a blog with Nuxt 3 and Directus step by step.
+
+[[toc]]
+
+### What is Nuxt 3?
+
+Nuxt 3 is an open source framework making web development simple and powerful. With Vue 3 and experience hybrid rendering, powerful data fetching and new features. [Visit the offical website](https://v3.nuxtjs.org).
+
+## Setup 🚀
+
+### 1. Create a new Nuxt 3 project
+
+```bash
+npx nuxi init my-blog
+
+cd my-blog/
+
+npm install
+```
+
+This created your Nuxt 3 application 💚.
+
+### 2. Create your Directus application [here](https://directus.cloud/login)
+
+### 3. Install the directus module & a CSS framework. We use TailwindCSS in this guide.
+
+```bash
+npm install nuxt-directus @nuxtjs/tailwindcss
+```
+
+### 4. Start the development server.
+
+```bash
+npm run dev
+```
+
+Your Nuxt 3 application is now running at <http://localhost:3000>.
+
+## Configuration ⚙️
+
+### 1. Update your `nuxt.config.ts`
+
+Change the URL to your Directus server in the configuration and add the modules we just installed. ✨
+
+```ts
+import { defineNuxtConfig } from "nuxt3";
+
+export default defineNuxtConfig({
+	modules: ["nuxt-directus", "@nuxtjs/tailwindcss"],
+	directus: {
+		url: "my-url",
+	},
+});
+```
+
+### Create a collection in the Directus admin dashboard 📄
+
+![image 1](1.webp)
```

Comment:
Lets use proper alt tags and file names for these images. For both A11y and SEO reasons 🙂

Reply:
Fixed with new commit